src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx

```diff
@@ -220,33 +220,33 @@ Labels are optional tags which help to identify collection data. Some examples a
 
     Example `kafka-config.yml` config file for this environment:
 
-```
```

Comment:
Unfortunately the site won't build with this change in place. I had to hand-edit this one for now. This should be fixed with the PR that recently got merged in MDX. For now, could you revert this change? I suspect this is what is causing the build to fail.

Reply:
yes I'll revert those changes!